juntagrico/lifecycle/sub.py

```diff
@@ -97,12 +97,16 @@ def check_sub_primary(instance):
             code='invalid')
     if instance.parts.count() > 0 and instance.future_parts.count() == 0 and instance.cancellation_date is None:
         raise ValidationError(
-            _('Nicht gekündigte {0} brauchen mindestens einen aktiven oder wartenden {0}-Bestandteil').format(
+            _('Nicht gekündigte {0} brauchen mindestens einen aktiven oder wartenden {0}-Bestandteil.'
+              ' Um die Kündigung rückgängig zu machen, leere und speichere zuerst das Kündigungsdatum des Bestandteils und dann jenes vom {0}.').format(
                 Config.vocabulary('subscription')),
             code='invalid')
 
 
 def check_children_dates(instance):
+    reactivation_info = ' Um die Aktivierung rückgängig zu machen oder in die Zukunft zu legen, ändere (bzw. leere) und speichere die Daten in dieser Reihenfolge:' \
```

Comment:
kannst du hier noch schnell den getttext wrapper rum machen?
und eventuell einen hint wenn man die aktivierung in die vergangenheit setzen will dass die schritte dan umgekehrt sind

Reply:
Scheint kein Problem zu machen, wenn das Datum in die Vergangenheit geändert wird. Das geht in einem Zug.